workflows-scripts/inventory/item-record-operations/README.md

```diff
@@ -0,0 +1,72 @@
+## Workflow steps:
+##### Workflow for item_recordOperations.jmx
+
```

Comment:
Add an introduction sentence or two, something like:
"This script contains 3 workflows to manage items: view, edit, and delete. The flows can be configured to control the probability of them being called by setting <some parameters>"

Reply:
Done